Formula/fruit.rb

```diff
@@ -0,0 +1,77 @@
+class Fruit < Formula
+  desc "Dependency injection framework for C++"
+  homepage "https://github.com/google/fruit/wiki"
+  url "https://github.com/google/fruit/archive/v3.1.0.tar.gz"
+  sha256 "d3307e02b8c85421290fc0c748ef2f964a9313596f963ae14176f35674d7230c"
+
```

Comment:
maybe add a link to `head` with `head "https://github.com/google/fruit.git"`?

Reply:
let's not